### ISSUE TITLE
XML import into DaVinci Resolve does not link audio with video

### ISSUE THREAD

**Issue by mckayc** [WyattBlue/auto-editor#368]
### Bug description

When you run auto-editor with `--export resolve` the resulting XML does not link audio and video when it is imported into DaVinci resolve. I have attached a gif to show what happens. This is similar to an issue from a couple years back #188 

Previously I was using `--export final-cut-pro` as a workaround but that no longer works with the latest version of auto-editor.

I am using DaVinci Resolve version 18.5.1 Build 6

![2023-08-16 -- 06 49 03 - Resolve_PH36fMfW1Z](https://github.com/WyattBlue/auto-editor/assets/3774563/0d295919-7ced-4c06-80bd-af6924e5f505)

### What's your platform?

```shell
Python Version: 3.11.4 64-bit
Platform: Windows 10 amd64 (Note; I am not sure why it returns this. I am on Windows 11)
FFmpeg Version: N-110322-g0c25da5a28-g1440bf15e2+1
FFmpeg Path: C:\Python311\Lib\site-packages\ae_ffmpeg\Windows\ffmpeg.exe
Auto-Editor Version: 23w32a
```

### What command did you use

```shell
auto-editor C:\myfile.mp4 --margin .05'sec' --export resolve --edit audio:threshold=4%
```

Thanks!

**Comment by WyattBlue** [comment]
Fixed by switching to the fcp11 backend

**Comment by mckayc** [comment]
Verified that this is now fixed on 23w33a-dev (23.33.1).
I am able to generate a resolve XLM and the resulting file links audio and video. Wahoo!

THANK YOU! This is a fantastic fix.

**Comment by cen0b** [comment]
Having this issue on the Auto Editor GUI, any idea when this fix will be pushed to the GUI?
